core/drivers/crypto/hisilicon/include/hpre_main.h

```diff
@@ -6,7 +6,8 @@
 #define _HPRE_MAIN_H
 
 #include <initcall.h>
-#include "hisi_qm.h"
+#include <rng_support.h>
+#include <hisi_qm.h>
```

Comment:
please sort in alphabetical order.

Note that this header fie does not need any of _initcall.h_ and _rng_support.h_. These header files should be included from the C-source files that need them. Instead, this header fie requires _stdint.h_ for sized types (`uint8_t`, `uint16_t` etc...)

Reply:
stdint.h is included in hisi_qm.h